### DISCUSSION TITLE
How to combine two phases (in order to plot a IPDF of combined crystal orientations)

### DISCUSSION THREAD

**Discussion by Arnknee** [mtex-toolbox/mtex#2010] [Ask Anything] [answered]
Hi

I have two phases, that have the same crystal symmetry.

I want to plot a IPDF with both phases included.

However I am only able to plot one.

Any ideas?

Code below of attempt to plot two phases.

```
ori = ebsd('Ti').orientations;

odf = calcODF(ori);

ori2 = ebsd('Tib').orientations;

odf2 = calcODF(ori2);

mtexFig = newMtexFigure('layout',[1,3],'figSize','huge');

nextAxis(1,1)

oM1.inversePoleFigureDirection=xvector;

plotIPDF(odf,xvector,'antipodal');

hold on;

plotIPDF(odf2,xvector,'antipodal');

hold off;

title( 'X(BD),Z(CS),Y [Tib-Ti]');

nextAxis(1,2)

oM1.inversePoleFigureDirection=zvector;

plotIPDF(odf,zvector,'antipodal');

hold on;

plotIPDF(odf2,zvector,'antipodal');

hold off;

nextAxis(1,3)

oM1.inversePoleFigureDirection=yvector;

plotIPDF(odf,yvector,'antipodal');

hold on;

plotIPDF(odf2,yvector,'antipodal');

hold off;

mtexColorbar
```

**Comment by kilir** [comment]
Hi,
if both are cubic, simply assign one symmetry to both:

```
ori = ebsd('Ti').orientations;
ori2 = ebsd('Tib').orientations;
ori2.CS = ori.CS;
odf = calcDensity([ori ori2]);
```
Hope that helps.

Cheers,
Rüdiger

**Reply by Arnknee** [reply]
Hi, thanks for the prompt reply.

I tried this and I didn't work, error **Error using cat
Dimensions of arrays being concatenated are not consistent.**

See phase details below

>> ebsd('Ti')
 
ans = EBSD
 
 Phase   Orientations  Mineral  Color  Symmetry  Crystal reference frame
     1  289216 (100%)       Ti      r     6/mmm       X||a*, Y||b, Z||c*
 
 Properties: bands, bc, bs, error, mad, reliabilityindex, x, y, grainId, mis2mean
 Scan unit : um
 cprInfo: show struct
 
>> ebsd('Tib')
 
ans = EBSD
 
 Phase   Orientations  Mineral  Color  Symmetry  Crystal reference frame
     0  146648 (100%)      Tib      g     6/mmm       X||a*, Y||b, Z||c*
 
 Properties: bands, bc, bs, error, mad, reliabilityindex, x, y, grainId, mis2mean
 Scan unit : um

**Reply by kilir** [reply, marked as answer]
Ok, in that case it's simply `[ori; ori2]` instead of `[ori ori2]` .

Anyhow, I somehow read that both of your phases cubic, however what you have there is hexagonal. In which case the above is only correct as long as they do not have different a/c ratios.

Cheers,
Rüdiger

**Reply by Arnknee** [reply]
Thanks, that worked!
